socketio.js

```diff
@@ -1,12 +1,17 @@
 const cookie = require("cookie");
 const request = require("superagent");
+const http2 = require("node:http2");
 
 const { get_conf, get_redis_subscriber } = require("./node_utils");
 const conf = get_conf();
 const log = console.log; // eslint-disable-line
 const subscriber = get_redis_subscriber();
 
-const io = require("socket.io")(conf.socketio_port, {
+const port = process.env.FRAPPE_SOCKETIO_SOCKET || process.env.PORT || conf.socketio_port;
```

Comment:
This (`FRAPPE_SOCKETIO_SOCKET`) allows me to namespace the socket per bench project and never worry about port conflicts again. One can argue, it's also slightly more secure as there is no way to accidentally bypass `nginx` which terminates the unix socket.

```
"/run/frappe/${benchProjectName}-socketio.socket"
```